src/MessagesCard/Completed.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import '../MessagesCard/styles/Style-MessagesCard.css'
+
+function Completed(){
+    return(
+        <div id="completed" className="completed">
+            <p>Successful payment</p>
+            <a href="/">Your shopping</a>
+        </div>
+    )
+}
```

Comment:
```suggestion
const Completed = () => (
    <div id="completed" className="completed">
        <p>Successful payment</p>
        <a href="/">Your shopping</a>
    </div>
)
```

The same for other simple components